index.html

```diff
@@ -73,7 +74,231 @@ <h3>LET'S CONNECT</h3>
+                <div class="get-resume-btn">
+                <input type="button" value="Get my resume" />
+                </div>
+            </div>
+
+            <div class="about-right">
+                <ul>
+                <li class="skills">
+                    <p><strong>Languages</strong></p>
+                    <i class="fa fa-chevron-down"></i>
+                </li>
+                <li class="skills skills-inner">
+                    <ul>
+                    <li>
+                        <img src="images/js.svg" />
+                        <p>JavaScript</p>
+                    </li>
+                    <li>
+                        <img src="images/html.svg" />
+                        <p>HTML</p>
+                    </li>
+                    <li>
+                        <img src="images/css.svg" />
+                        <p>CSS</p>
+                    </li>
+                    </ul>
+                </li>
+                <li class="skills">
+                    <p><strong>Frameworks</strong></p>
+                    <i class="fa fa-chevron-right"></i>
+                </li>
+                <hr />
+                <li class="skills">
+                    <p><strong>Skills</strong></p>
+                    <i class="fa fa-chevron-right"></i>
+                </li>
+                <hr />
+                </ul>
+            </div>
+            </section>
+
+            <!-- End: About Section-->
+
+            <!-- Start: Contact Form-->
+            <section id="contact-section" class="form-holder">
+            <div class="form-wrapper">
+                <div class="form-description">
+                <h2>Contact me</h2>
+                <p>
+                    If you have an application you are interested in developing, a
+                    feature that you need built or a project that needs coding. I’d love
+                    to help with it!
+                </p>
+                </div>
+                <form action="https://formspree.io/f/myyaorpl" method="post">
+                <fieldset>
+                    <input
+                    type="text"
+                    name="user_name"
+                    placeholder="Lorem Ipsum"
+                    maxlength="30"
+                    required
+                    />
+                    <input
+                    type="email"
+                    name="email"
+                    placeholder="example@example.com"
+                    required
+                    />
+                    <textarea
+                    maxlength="500"
+                    name="message"
+                    placeholder="Write your message here..."
+                    required
+                    ></textarea>
+                    <button type="submit">Get in touch</button>
+                </fieldset>
+                </form>
+                <div class="finish-line"></div>
+            </div>
+        </section>
+
+        <!-- End: Contact Form-->
```

Comment:
- [ ] The contact section is another milestone by itself, so I advise you to remove this code and save it somewhere because you will use it in the next milestone.